.vscode/launch.json

```diff
@@ -4,6 +4,18 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        {
+            "name": "Python: FastAPI",
```

Comment:
Could replace the `Worker Service` with this, and also rename it to `Worker Service`